src/main/java/org/tarantool/jdbc/type/TarantoolSqlType.java

```diff
@@ -10,33 +10,45 @@
 
     UNKNOWN(TarantoolType.UNKNOWN, JdbcType.UNKNOWN, "unknown"),
 
+    // float, double, real used to be number aliases before 2.2
```

Comment:
I guess tarantool will support 'float32' / 'float64' (don't sure about names) natively in a future and they will have corresponding FLOAT/DOUBLE/REAL aliases in Tarantool/SQL. After that it maybe will not be appropriate to map them here to 'number', so this mapping will not be static anymore. However it is correct for now and maybe it will be possible to make this mapping server versions dependent, so let it be. Just noted my doubts.